BuildLinux.sh

```diff
@@ -166,6 +166,7 @@ then
         # have to build deps with debug & release or the cmake won't find evrything it needs
```

Comment:
Just notced this little typo. 😊
```suggestion
        # have to build deps with debug & release or the cmake won't find everything it needs
```